content/millennium/r4/proprietary-codes-and-systems.md

```diff
@@ -793,6 +799,15 @@ This system is the account number of a financial account.
       "value": "5646"
     }
 
+##### Bill Codes Types
+
+This system is the bill code type for a charge item.`Bill code type` can be either `CDM_SCHED`, `CPT`, `HCPCS`, or `REVENUE`.
+
+    {
+      'system': 'https://fhir.cerner.com/<Ehr source id>/CodeSystem/BillCodes-<Bill code type>',
+      "code": "0310"
```

Comment:
nit - single quotes instead of double quotes on this line

Reply:
Almost all the json in this file have double-quotes for some reason. Updated system for uniformity [here](https://github.com/cerner/fhir.cerner.com/pull/605/commits/61aa81539c2b29d67f02a8bb87769033ab4cb6bb#diff-aedba7f37b773d7a314400db492774b8a9d406ed43aad2101793b82970106103R807)